src/main/java/uk/gov/hmcts/reform/cwrdapi/repository/SkillRepository.java

```diff
@@ -1,12 +1,21 @@
 package uk.gov.hmcts.reform.cwrdapi.repository;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 import uk.gov.hmcts.reform.cwrdapi.domain.Skill;
 
+import java.util.List;
+
 @Repository
-public interface SkillRepository extends JpaRepository<Skill,Long> {
+public interface SkillRepository extends JpaRepository<Skill, Long> {
+
 
+    @Query(value = """
```

Comment:
""" - reformat this to "

Reply:
As we are using java 17 , considering that using textblock instead of string concatenation.